src/components/wordInput.tsx

```diff
@@ -206,6 +220,13 @@ export const WordInput = forwardRef<HTMLInputElement, WordInputProps>((props, in
               class: "actionButton",
             },
           ]}
+          buttonAttributes={[
+            {
+              attribute: "disabled",
+              value: "true",
+              buttons: disabledLetters.replace(/./g, '$& '),
```

Comment:
simple-keyboard needed spaces between each letter in order to parse this string

Reply:
Ah, got it; this works as well